data_model/src/domain.rs

```diff
@@ -73,8 +72,6 @@ mod model {
     pub struct Domain {
         /// Identification of this [`Domain`].
         pub id: DomainId,
-        /// [`Account`]s of the domain.
-        pub accounts: AccountsMap,
         /// [`Asset`](AssetDefinition)s defined of the `Domain`.
         pub asset_definitions: AssetDefinitionsMap,
```

Comment:
We might consider excluding `asset_definitions` as well (probably in separate PR).

Reply:
separate issue for sure. This was POC